packages/dialect-react/api/web2.ts

```diff
@@ -30,6 +30,56 @@ const signPayload = async (wallet: WalletContextState, payload: Uint8Array) => {
   };
 };
 
+const generateToken = async (wallet: WalletType): Promise<TokenType> => {
+  const tokenTTLMinutes = 180;
+  const now = new Date().getTime();
+
+  const expirationTime = now + tokenTTLMinutes * 60000;
+  const dateEncoded = new TextEncoder().encode(
+    btoa(JSON.stringify(expirationTime))
+  );
+
+  const { signature } = await signPayload(
+    wallet as WalletContextState,
+    dateEncoded
+  );
+
+  const base64Signature = btoa(
+    String.fromCharCode.apply(null, signature as unknown as number[])
+  );
+
+  return {
+    expirationTime: expirationTime.toString(),
+    base64Signature
+  };
+}
+
+const saveToken = (token: TokenType) => {
+  window.sessionStorage.setItem("expirationTime", token.expirationTime.toString());
```

Comment:
Just realized that this is going to break in cases where a project is using Nextjs SSR. Let's wrap with a small check that `window` exists or similar. wdyt?

Reply:
Also, quick question, why not wrap these 2 entries into one, assuming they are paired together.